src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -325,6 +326,56 @@ void CheckPReLUGradientCorrect(const arma::colvec input,
   BOOST_REQUIRE_CLOSE(gradient(0), target(0), 1e-3);
 }
 
+/*
+ * Simple SELU activation test to check whether the mean and variance remain
+ * invariant after passing through the function.
+ *
+ * NOTE: the input must be normalized.
+ */
+
+BOOST_AUTO_TEST_CASE(SELUFunctionTest)
+{
+  arma::mat input = arma::randn<arma::mat>(1000, 1);
+
+  arma::mat output;
+
+  SELUFunction::Fn(input, output);
+
+  BOOST_REQUIRE_LE(
+     arma::as_scalar(arma::abs(arma::mean(input) - arma::mean(output))), 0.01);
+
+  BOOST_REQUIRE_LE(
+     arma::as_scalar(arma::abs(arma::var(input) - arma::var(output))), 0.01);
+}
```

Comment:
Would it make sense to also test with data that is not already normalized, and ensure that the output is changed sufficiently?

Reply:
Yes, I can add a test in which the mean and variance of non-normalized inputs do not remain the same after passing through the SELU function.